### DISCUSSION TITLE
Using a machine name other than RDPWindows

### DISCUSSION THREAD

**Discussion by greatawesome** [winapps-org/winapps#356] [Support] [answered]
Is it possible to use a different Windows machine name? If so, what would need to be changed to make that work? 

Edit: I should note that I'd be looking to use KVM/libvirt since I need to forward some physical hardware to the VM, and I don't know if that would be feasible in Docker.

**Comment by oskardotglobal** [comment, marked as answer]
No, that is not possible at the moment.
